jellyfin-helpers/jellyfin_helpers/workout_plan/plan_workouts.py

```diff
@@ -197,46 +231,54 @@ def create_workout_plan(
                         "tool": [""],
                     }
                 )
+                plan = pd.concat([plan, new_row])
             else:
-                selected_workouts = self._search_for_workout(
-                    row=row,
-                    skip_ids=all_skip_ids
-                    # currently not enough entries for these filters
-                    if row["values"]
-                    not in [
-                        "tennis",
-                        "mobility",
-                        "calves",
-                        "stretching",
-                        "stretch/back",
-                        "dance single",
-                    ]
-                    # TODO would ideally not want the same one in a week
-                    # as jellyfin does not add duplicates to playlist
-                    else [],
-                )
-                item_ids = selected_workouts.id.values
-                in_month_skip_ids.extend(item_ids)
-                all_skip_ids.extend(item_ids)
-
-                num_entries = selected_workouts.shape[0]
-                new_row = pd.DataFrame(
-                    {
-                        "week": [week] * num_entries,
-                        "day": [days_from_now] * num_entries,
-                        "source_type": ["video"] * num_entries,
-                        "key": [key] * num_entries,
-                        "duration_in_min": selected_workouts.duration.apply(
-                            lambda x: x.seconds // 60
-                        ).values,
-                        "optional": [row.optional] * num_entries,
-                        "time_of_day": [row.time_of_day] * num_entries,
-                        "item_id": item_ids,
-                        "description": selected_workouts.description.values,
-                        "tool": selected_workouts.tool.values,
-                    }
-                )
-
-            plan = pd.concat([plan, new_row])
+                try:
+                    selected_workouts = self._search_for_workout(
+                        row=row,
+                        skip_ids=all_skip_ids
+                        # currently not enough entries for these filters
+                        if row["values"]
+                        not in [
```

Comment:
since logs a warning now, do we want to remove these exceptions so that we start to manually address these issues?